internal/model/modelindexer/indexer.go

```diff
@@ -444,7 +539,109 @@ func (i *Indexer) runActiveIndexer() {
 				return err
 			})
 		}
+		if i.config.Scaling.Disabled {
+			continue
+		}
+		now := time.Now().Add(time.Duration(id))
```

Comment:
Why is `id` being added to the time?

Reply:
Yep, you're right. I've implemented what you suggest in the latest commit.

Reply:
Thanks, looks good.